subjects/project_motion/README.md

```diff
@@ -24,17 +35,25 @@ You must implement :
 
 #[derive(Debug, Clone, PartialEq)]
 pub struct Object {
-    pub distance: f32,
-    pub velocity: f32,
-    pub height: f32,
+    pub x: f32,
+    pub y: f32,
+}
+
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
do not know, but could be nice to change the name to `Point` or `coordinate`? let me know what you think